### PR TITLE
Added a grammar mistake

### PR DESCRIPTION
**Pull request by facusapienza21**

### CODE REVIEW

File: agu_oss/clean_data.py [hunk 1]
```diff
@@ -33,6 +33,6 @@ def open_and_clean(url):
     temp_df = temp_df.assign(Day=1)
 
     # Finally create a date time column
-    temp_df["date"] = pd.to_datetime(temp_df[['Year', 'Month', 'Day']])
+    temp_df["date"] = pd.to_dddddddddddddddatetime(temp_df[['Year', 'Month', 'Day']])
```
Comment (lwasser) [inline agu_oss/clean_data.py:36]: @facusapienza21 did you really mean to misspell this ??

Comment (facusapienza21) [reply agu_oss/clean_data.py:36]: That was exactly my intention :)

Comment (lwasser) [reply agu_oss/clean_data.py:36]: nice work in breaking things @facusapienza21 ! :) 🚀

Comment (Cecile-A-C) [reply agu_oss/clean_data.py:36]: I didn't know we could comment each other's pull requests!

Comment (lwasser) [reply agu_oss/clean_data.py:36]: pretty, cool, eh @Cecile-A-C !? you can comment on any open  pull requests  which really makes collaboration and peer review effective on github!

**Pull request closed without merging**